src/game.js

```diff
@@ -281,6 +281,10 @@
                 me.input._updateGamepads();
 
                 accumulator += me.timer.getDelta();
```

Comment:
would probably be interesting though to understand why `getDelta()` returns a negative value...

Reply:
I am confused. But I debug it with Wechat game dev tool, from its console, I found it negative